src/pages/cart/CartPage.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import Layout, { Composition, Box, Only } from 'atomic-layout'
+import styled from 'styled-components'
+import { IoMdClose } from 'react-icons/io'
+import { Heading } from '../../atoms/Heading'
+import { Grid } from '../../atoms/Grid'
+import { IoIosArrowForward } from 'react-icons/io'
```

Comment:
You require `react-icons/io` module twice. Please merge the imports into a single one.